src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1007,6 +1033,20 @@ class Command(Action):
 
     def __call__(self, **kwds):
         """Call a query with the specified keyword arguments."""
+        if hasattr(self, "file_name"):
+            file_purpose = self.file_name.get_attr(_InlineConstants.file_purpose)
+        elif hasattr(self, "file_name_list"):
+            file_purpose = self.file_name_list.get_attr(_InlineConstants.file_purpose)
+        elif hasattr(self, "filename"):
+            file_purpose = self.filename.get_attr(_InlineConstants.file_purpose)
+        print(f"\nfile_purpose = {file_purpose}\n")
+
+        if file_purpose == "input" and pypim.is_configured():
```

Comment:
I believe we should not be adding such specific code in `flobject.py`, can we hook or wrap around the `file_purpose` attribute at a higher level?

Reply:
We are using remote file handler's method now.